.github/workflows/wordpress-org-screenshots.yml

```diff
@@ -34,7 +34,8 @@ jobs:
           'mr',
           'nl_BE',
           'it_IT',
-          'en_US'
+          'en_US',
+          'pt_BR'
```

Comment:
Not a blocker, but let's put these in alphabetical order when we have a moment. As this list grows it will be easier to find the different locales.